services/eventServices.js

```diff
@@ -0,0 +1,28 @@
+var Event = require('../models/event.js');
+var moment = require('moment');
+
+var eventServices = {};
+
+eventServices.createEvent = function(eventData, cb) {
+    var newEvent = eventData;
+    console.log(newEvent);
+    newEvent.startDate = moment(newEvent.startDate).toDate();
+    Event.create(newEvent, function(err, event) {
+        cb(err, event);
+    });
+};
+
+eventServices.findOneEvent = function(eventData, cb) {
```

Comment:
you should pass the id as an argument here, not all of the event data. the function will work either way, but it's better practice to only pass what's necessary.